src/covid_model_seiir_pipeline/ode_fit/model.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
```

Comment:
Straight port of https://github.com/ihmeuw/covid-model-seiir/blob/develop/src/covid_model_seiir/ode_model/ode_process.py